engines/sci/graphics/controls32.h

```diff
@@ -108,6 +200,12 @@ class GfxControls32 {
 
 	reg_t kernelEditText(const reg_t controlObject);
 
+	reg_t registerScrollWindow(ScrollWindow *scrollWindow);
```

Comment:
Can these signatures be moved together so that each feature is grouped, as it is done in `frameout.h`?